var/spack/repos/builtin/packages/openradioss-engine/package.py

```diff
@@ -54,26 +54,26 @@ class OpenradiossEngine(CMakePackage):
     @property
     def compiler_name(self):
         compiler_mapping = {
-            "aocc": "64_AOCC",
-            "intel": "64_intel",
-            "oneapi": "64_intel",
-            "gcc": "64_gf",
-            "arm": "a64_gf",
+            "aocc": "linux64_AOCC",
+            "intel": "linux64_intel",
+            "oneapi": "linux64_intel",
+            "gcc": "linux64_gf",
+            "arm": "linuxa64",
         }
         compiler_name = compiler_mapping[self.spec.compiler.name]
         return compiler_name
 
     def cmake_args(self):
         args = [
             "-Dmpi_os=0",
-            "-DCMAKE_Fortran_COMPILER={0}".format(spack_fc),
-            "-DCMAKE_C_COMPILER={0}".format(spack_cc),
-            "-DCMAKE_CPP_COMPILER={0}".format(spack_cxx),
-            "-DCMAKE_CXX_COMPILER={0}".format(spack_cxx),
+            f"-DCMAKE_Fortran_COMPILER={spack_fc}",
+            f"-DCMAKE_C_COMPILER={spack_cc}",
+            f"-DCMAKE_CPP_COMPILER={spack_cxx}",
+            f"-DCMAKE_CXX_COMPILER={spack_cxx}",
```

Comment:
Can you use `self.define` here and in general in the recipe, since you are updating style? That would be:
```python
self.define("CMAKE_Fortran_COMPILER", spack_fc)
```

Reply:
I have committed again with a changed style.